### PR TITLE
fix width/height issues with run and exec

### PR DESCRIPTION
**Pull request by ddollar**
fixes strange wrapping issues with run and exec

## Release Playbook
- [x] Rebase against master
- [x] Pass checks
- [x] Release branch (20160225210003-tty-size)
- [x] Pass CI (https://circleci.com/gh/convox/rack/585)
- [x] Code review
- [ ] Merge into master
- [ ] Release master ()
- [ ] Pass CI ()
- [ ] Update dev and testing racks
- [ ] If rack or app formation changes, test upgrade → re-deploy existing app → downgrade cycle
- [ ] Publish release
- [ ] Release CLI

### CODE REVIEW

**Comment by nzoschke** [comment]
Do height and width need defaults for old clients?

Otherwise :+1:

File: api/models/app.go [hunk 1]
```diff
@@ -403,6 +405,14 @@ func (a *App) ExecAttached(pid, command string, rw io.ReadWriter) error {
 	go io.Copy(iw, rw)
 	go io.Copy(rw, or)
 
+	success := make(chan struct{})
+
+	go func() {
+		<-success
+		d.ResizeExecTTY(id, height, width)
```
Comment (mattmanning) [inline api/models/app.go]: Will this TTY setting persist beyond the life of this goroutine? If so any side effects to consider?

**Comment by mattmanning** [comment]
Looks good. Straightforward and easy to understand. :+1:

**Comment by ddollar** [comment]
No. The `strconv.Atoi` fails on the empty string and returns 0 which works as a default value.

File: api/models/app.go (hunk 1 above)
Comment (ddollar) [inline api/models/app.go]: It's resizing the TTY on the docker daemon end so should be fine.

**Pull request merged**